server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -54,11 +54,11 @@ import {
     BaseTelemetryProperties,
     SessionState,
 } from "@fluidframework/server-services-telemetry";
-import { setQueuedMessageProperties } from "../utils";
+import { DocumentContext } from "@fluidframework/server-lambdas-driver";
+import { setQueuedMessageProperties ,logCommonSessionEndMetrics, createSessionMetric } from "../utils";
 import { CheckpointContext } from "./checkpointContext";
```

Comment:
super minor nit: space should come after comma, not before in ` ,logCommonSessionEndMetrics`